GameOfLifeConsole/Program.cs

```diff
@@ -0,0 +1,136 @@
+                    {0,0,0,0,0},
+                    {0,0,0,0,0}
+                };
+
+                    GameSeed gameSeed = new GameSeed(seed);
+
+                    Console.ReadLine();
+                    Console.WriteLine("\n Start Field");
+
+                    gameSeed.DrawField();
+
+                    Console.WriteLine();
+
+                    while (gameSeed.AliveCells() > 0 && runs++ < MaxRuns)
+                    {
+
+                        Console.WriteLine();
+
+                        gameSeed.NewCellGeneration();
+                        gameSeed.DrawField();
+
+                        Console.WriteLine();
+
+                        if (gameSeed.AliveCells() == 0)
+                        {
+                            Console.WriteLine("Everyone is died!");
+                            Console.ReadLine();
+                        }
+                        else
+                        {
+                            Console.ReadLine();
+
+                        }
+                    }
+                    break;
+                
+                case ConsoleKey.R:
+                    while (true)
+                    {
+                        Console.Clear();
+
+                        try
+                        {
+                            Console.WriteLine("Please, insert the size of game and number of cells. \t");
+
+                            Console.Write("The number of rows:\t ");
+                            int row = int.Parse(Console.ReadLine());
+                            Console.Write("The number of columns:\t ");
+                            int column = int.Parse(Console.ReadLine());
+
+                            int[,] gameField = new int[row, column];
+                            Random random = new Random();
+
+                            for (int y = 0; y < gameField.GetLength(0); y++) // Solid principles, the code of random numbers generating.
+                            {
+                                for (int x = 0; x < gameField.GetLength(1); x++)
+                                {
+                                    gameField[y, x] = random.Next(2); // 0- dead cell, 1- alive
+                                }
+                            }
+
+                            for (int y = 0; y < gameField.GetLength(0); y++) // The part which input the array elements.
+                            {
+                                for (int x = 0; x < gameField.GetLength(1); x++)
+                                {
+
+                                }
+                                Console.WriteLine();
+                            }
+
+                            GameSeed gameSeedRandom = new GameSeed(gameField);
+
+                            Console.WriteLine("Field 0");
+                            gameSeedRandom.DrawField();
+                            Console.WriteLine();
+
+                            while (gameSeedRandom.AliveCells() > 0 && runs++ < MaxRuns)
```

Comment:
This same exact while loop repeats itself two times in this file.
Try to refactor this so it's here only one.

Reply:
Little hint so you don't go in wrong direction - you don't need new methods for that.

Reply:
Done.